homeassistant/components/synology_dsm/const.py

```diff
@@ -247,5 +245,27 @@
     },
 }
 
+INFORMATION_SENSORS = {
+    f"{SynoDSMInformation.API_KEY}:temperature": {
+        ENTITY_NAME: "temperature",
+        ENTITY_UNIT: None,
+        ENTITY_ICON: "mdi:thermometer",
```

Comment:
Entities with device class set should not set a custom icon.

Reply:
should I update the whole file or only what I added ?

Reply:
If it's not too much, it's nice to clean the whole file. But it can also be done in another PR.